### DISCUSSION TITLE
username_filter not working

### DISCUSSION THREAD

**Discussion by jsabia85** [Taxel/PlexTraktSync#1231] [Support] [answered]
I am trying to get the username_filter option to work but it keeps syncing all users who are invited to my plex watch history.

in the config.yml I have the following and also ran plextraktsync watch

```yml
watch:
  username_filter: true
```

**Comment by glensc** [comment]
pay attention that username_filter is under watch section, meaning it is for watch command not for sync command.

**Reply by jsabia85** [reply]
I have it under the watch command and not sync.

**Reply by glensc** [reply, marked as answer]
that's exactly what I said, it's for watch command.

**Comment by glensc** [comment]
Perhaps it's the same bug:
- https://github.com/Taxel/PlexTraktSync/issues/1339

create a new bug report of your setup with full details., answer the same questions that the user there failed to answer and provide the necessary details.
